Modules/_elementtree.c

```diff
@@ -813,12 +814,16 @@ _elementtree_Element___deepcopy___impl(ElementObject *self, PyObject *memo)
 
     PyTypeObject *tp = Py_TYPE(self);
     elementtreestate *st = get_elementtree_state_by_type(tp);
-    tag = deepcopy(st, self->tag, memo);
+    tmp = Py_NewRef(self->tag);
+    tag = deepcopy(st, tmp, memo);
+    Py_CLEAR(tmp);
     if (!tag)
         return NULL;
 
     if (self->extra && self->extra->attrib) {
-        attrib = deepcopy(st, self->extra->attrib, memo);
+        tmp = Py_NewRef(self->extra->attrib);
```

Comment:
This will kill the optimization in `deepcopy()`. Only increment the refcount after checking the fast path.